.github/workflows/telemetry-ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   build-and-test:
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        os: macos-latest
```

Comment:
CI needs to run on all those platforms.

Reply:
Applies to all the packages that can run on windows and linux. I just spent a good amount of time hardening and fixing issues that were a result of not consistently testing multiple platforms.

Reply:
Reverted.  Had a brain fart around CI needing to run the other platforms.